css/src/tokens/themes.scss

```diff
@@ -42,15 +42,17 @@ $themes: (
 		'score-medium': $palette-yellow-50,
 		'score-high-off': $palette-green-opacity-30,
 		'score-high': $palette-green-50,
-		'hyperlink': $palette-blue-60,
-		'primary-base': $palette-blue-50,
+		'hyperlink': $palette-blue-60-deprecated,
+		'primary-base': $palette-blue-110,
 		'primary-background': $palette-blue-10,
-		'primary-background-glow-high-contrast': $palette-blue-10,
-		'primary-dark': $palette-blue-80,
-		'primary-hover': $palette-blue-60,
-		'primary-active': $palette-blue-70,
+		'primary-background-glow-high-contrast': $palette-blue-10-deprecated,
+		'primary-dark': $palette-blue-120,
+		'primary-hover': $palette-blue-120,
+		'primary-active': $palette-blue-170,
 		'primary-box-shadow': $palette-blue-opacity-30,
 		'primary-invert': $palette-white,
+		'primary-hover-light-background': $palette-blue-20,
```

Comment:
Added this to introduce a new color for tags. I'm not confident that the other semantic colors will also need such item, but we will need to add it to other colors make the color index in colors.scss consistent.

Reply:
Perhaps you could outline why this was added? Looking though both the badge, button, and tag. I don't see why we need this one when we have the current -hover value.

I do see the background color in tag seems to be very light (pallete-10), but I don't see the "X" button having any hover styles at all.

https://react.fluentui.dev/?path=/docs/components-tag-tag--docs

Reply:
Each tag dismiss button has 115ea3 on hover (the color that I've set for `primary-dark` and `primary-hover`, it's just really hard to see. It corresponds to the `colorCompoundBrandForeground1Hover` Fluent color token.

![image](https://github.com/user-attachments/assets/6a88d7a6-846b-42d4-b0e0-e82c1c65946f)

![image](https://github.com/user-attachments/assets/a37eeb47-0eee-4d7d-9e89-ee9ac065ee50)

`primary-base/hover` values were updated to match primary buttons, hence no adjustments are needed for buttons. It still only uses two blue colors. 

For Atlas tag colors, we've kept it simple by using three blue colors (primary-dark on hover). 

`primary-hover-light-background` was added for interactive tags. On hover, Fluent uses a different blue that's not already included in primary-base, dark, nor hover values. There's a total of four blue colors.

![image](https://github.com/user-attachments/assets/b445ecc6-8e04-4eee-9202-5157e9adfe55)

![image](https://github.com/user-attachments/assets/4714e77b-37d5-4d06-bcb5-cd790366de34)

A pattern that I've noticed is that Fluent can have multiple combos for the same light theme color, but different color for dark and/or high contrast.

Another pattern is using more than one set of background/foreground/whatever colors. Having just one set of `primary-background` colors may not fit all of the use cases.

Reply:
A pattern that I've noticed is that Fluent can have multiple combos for the same light theme color, but different color for dark and/or high contrast.

Another pattern is using more than one set of background/foreground/whatever colors. Having just one set of `primary-background` colors may not fit all of the use cases. 

![image](https://github.com/user-attachments/assets/08a11856-5bd8-4d7b-9eb0-7e0abf35089c)

Reply:
Renamed to `primary-background-hover`